pages/stats.js

```diff
@@ -82,15 +83,21 @@ const SortableHeader = ({
   );
 };
 
-const StatsPage = ({ stats }) => {
+const StatsPage = ({ stats, recentStats }) => {
+  const [order, setOrder] = useState('desc');
+  const [selectedSortKey, setSelectedSortKey] = useState('avgScore');
+  const [statsRange, setStatsRange] = useState('recent');
+
+  let relevantStats;
+  if (statsRange === 'recent') relevantStats = recentStats;
+  if (statsRange === 'allTime') relevantStats = stats;
+
```

Comment:
Constants for `'recent'` and `'allTime'`? (Avoid hard-coding twice.)

Also, I'd just use a `const` + ternary to avoid the braces-less if-statements:
```suggestion
  const relevantStats = (statsRange === 'recent') ? recentStats : stats;
```